modin/pandas/test/test_io.py

```diff
@@ -200,6 +201,22 @@ def eval_to_file(modin_obj, pandas_obj, fn, extension, **fn_kwargs):
         teardown_test_files([unique_filename_modin, unique_filename_pandas])
 
 
+@pytest.fixture
+def make_parquet_dir():
+    path: str = ""
+
+    def _make_parquet_dir(
+        dfs_by_filename: Dict[str, pandas.DataFrame], row_group_size: int
+    ):
+        for filename, df in dfs_by_filename.items():
+            df.to_parquet(os.path.join(path, filename), row_group_size=row_group_size)
+        return path
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        path = tmp_dir
```

Comment:
Won't this override the `path` arg specified when calling `make_parquet_dir`?

Reply:
Yes, that's what I want to happen. It's kind of hacky but I'm doing this on purpose to set the path to the tmp_dir path, and allow the context manager to delete the directory after we return from the function yield.

Reply:
I would rather you declare `path` and `_make_parquet_dir()` under the same context manager, as right now this closure looks strange (it might work given how closures compute their cell variables, but it still reads really weird):
```python
with tempfile.TemporaryDirectory() as path:
    def _make_parquet_dir(....):
        for filename, ....:
            df.to_parquet(os.path.join(path, filename), ....)
    yield _make_parquet_dir
```